drivers/amazonec2/amazonec2.go

```diff
@@ -36,7 +36,7 @@ const (
 	machineTag                  = "rancher-nodes"
 	defaultAmiId                = "ami-c60b90d1"
 	defaultRegion               = "us-east-1"
-	defaultInstanceType         = "t2.micro"
+	defaultInstanceType         = "t3.micro"
```

Comment:
Does every region have t3.micro ? Worried we run into opposite problem where something that did work in a different region now fails because it only has t2 and not t3.

Reply:
@cbron I have not been able to find straight forward documentation for this; we may have to test it. It is stated that t3.micro is only in the free tier where t2.micro is unavailable: https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc&awsf.Free%20Tier%20Types=*all&awsf.Free%20Tier%20Categories=*all.

Reply:
It's possible that the error is valid and they simply must change the instance type accordingly.

Reply:
Without knowing which ones are valid where, I don't want to blindly change it, too high of a risk of breaking existing setups. t2 being the free one means its more likely to be available in my mind. Is there a way we can check with the api or websites ?

Reply:
If changing the default is too high of a risk, we could add a set of regions where t2 is unavailable and change the default to t3 based on the region before hitting the aws API to create the instance.

Not sure where this code would go exactly (probably inside `innerCreate()`) , but this is the idea.
```
regionsWithoutT2 := map[string]struct{}{
	"me-south-1": struct{}{},
}

regionToCheck := "me-south-1"
if _, ok := regionsWithoutT2[regionToCheck]; ok {
	// use t3.micro as default
} else {
        // use t2.micro as default
}
```